docs/test-world-2/launching-a-node.mdx

```diff
@@ -27,7 +27,7 @@ The Testworld Mission 2.0 network:
 The `seeds.txt file` location: 
```

Comment:
sholdn't this file be deleted too ... all the testworld section should have gone I think